models/articles-model.js

```diff
@@ -35,10 +43,25 @@ function selectArticles(){
     articles.votes,
     articles.article_img_url
     ORDER BY
-    articles.created_at DESC;`)
+    articles.created_at DESC`
+
+  let queryValues;
+
+    if (topic) {
```

Comment:
I think we should check beforehand if the topic exists - perhaps by using a greenlist of all the topics in the database that can be updated dynamically? 😄